sdk/metric/meter_test.go

```diff
@@ -532,6 +545,106 @@ func TestMeterCreatesInstruments(t *testing.T) {
 	}
 }
 
+func TestMeterWithDropView(t *testing.T) {
+	testCases := []struct {
+		name string
+		fn   func(*testing.T, metric.Meter)
+	}{
+		{
+			name: "SyncInt64Count",
+			fn: func(t *testing.T, m metric.Meter) {
+				ctr, err := m.Int64Counter("sint")
+				assert.NoError(t, err)
+				c, ok := ctr.(x.EnabledInstrument)
+				assert.True(t, ok)
```

Comment:
```suggestion
				require.True(t, ok)
```

otherwise the next line will panic